web_company_color/models/res_company.py

```diff
@@ -98,6 +98,7 @@ class ResCompany(models.Model):
     color_button_bg_hover = fields.Char(
         "Button Background Color Hover", sparse="company_colors"
     )
+    color_input_bg = fields.Char("Input Background Color", sparse="company_colors")
```

Comment:
```suggestion
    color_required_input_bg = fields.Char("Required Input Background Color", sparse="company_colors")
```

Reply:
thanks @Tardo , addressed in fixup commit.